src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -817,7 +817,7 @@ private unsafe List<string> AddDatabase(TransactionOperationContext context, Bli
                 var key = $"{Helpers.ClusterStateMachineValuesPrefix(databaseName)}{keyValue.Key}";
                 using (Slice.From(context.Allocator, key, out Slice databaseValueName))
                 using (Slice.From(context.Allocator, key.ToLowerInvariant(), out Slice databaseValueNameLowered))
-                using (var value = EntityToBlittable.ConvertCommandToBlittable(keyValue.Value, context))
+                using (var value = context.ReadObject(keyValue.Value, key))
```

Comment:
use `Clone` method from BJRO

Reply:
Actually why we need to re-read the blittable here? It is on a different context? I do not think so?

Reply:
IIRC it throws with something. I will remove this and let you know or I will give a try with clone method.

Reply:
When I get rid of read object:

```
2019-02-16T09:59:39.0297239Z, 22, Information, A, Raven.Server.Rachis.RachisConsensus, Error when running leader behavior, EXCEPTION: System.InvalidOperationException: Attempted to copy a nested object. This will actually copy the whole object, which is probably not what you wanted.
   at Sparrow.Json.BlittableJsonReaderObject.InvalidAttemptToCopyNestedObject() in C:\workspaces\ravendb\src\Sparrow\Json\BlittableJsonReaderObject.cs:line 872
   at Sparrow.Json.BlittableJsonReaderObject.get_BasePointer() in C:\workspaces\ravendb\src\Sparrow\Json\BlittableJsonReaderObject.cs:line 173
   at Raven.Server.ServerWide.ClusterStateMachine.UpdateValue(Int64 index, Table items, Slice lowerKey, Slice key, BlittableJsonReaderObject updated) in C:\workspaces\ravendb\src\Raven.Server\ServerWide\ClusterStateMachine.cs:line 755
   at Raven.Server.ServerWide.ClusterStateMachine.SetDatabaseValues(Dictionary`2 databaseValues, String databaseName, TransactionOperationContext context, Int64 index, Table items) in C:\workspaces\ravendb\src\Raven.Server\ServerWide\ClusterStateMachine.cs:line 821
   at Raven.Server.ServerWide.ClusterStateMachine.AddDatabase(TransactionOperationContext context, BlittableJsonReaderObject cmd, Int64 index, Leader leader) in C:\workspaces\ravendb\src\Raven.Server\ServerWide\ClusterStateMachine.cs:line 788
   at Raven.Server.ServerWide.ClusterStateMachine.Apply(TransactionOperationContext context, BlittableJsonReaderObject cmd, Int64 index, Leader leader, ServerStore serverStore) in C:\workspaces\ravendb\src\Raven.Server\ServerWide\ClusterStateMachine.cs:line 275
   at Raven.Server.Rachis.RachisStateMachine.Apply(TransactionOperationContext context, Int64 uptoInclusive, Leader leader, ServerStore serverStore, Stopwatch duration) in C:\workspaces\ravendb\src\Raven.Server\Rachis\RachisStateMachine.cs:line 45
   at Raven.Server.Rachis.RachisConsensus`1.Apply(TransactionOperationContext context, Int64 uptoInclusive, Leader leader, Stopwatch duration) in C:\workspaces\ravendb\src\Raven.Server\Rachis\RachisConsensus.cs:line 74
   at Raven.Server.Rachis.Leader.OnVoterConfirmation() in C:\workspaces\ravendb\src\Raven.Server\Rachis\Leader.cs:line 472
   at Raven.Server.Rachis.Leader.Run() in C:\workspaces\ravendb\src\Raven.Server\Rachis\Leader.cs:line 321
2019-02-16T09:59:39.0336249Z, 68, Information, Server, Raven.Server.RavenServer, TCP 
```

Reply:
Force pushed change with clone method. Thanks for suggestion.